src/Rokt-Kit.js

```diff
@@ -22,6 +22,9 @@ function attachLauncher(accountId, sandboxMode) {
     window.Rokt.createLauncher({
         accountId: accountId,
         sandbox: sandboxMode,
+        integrationName: process.env.PACKAGE_VERSION
```

Comment:
If I'm reading this right, the `integrationName` will just include the version of the Kit, correct?  I think there might be some value in collecting the version of the Core SDK as well, in case we notice some compatibility issues between the two.

You can grab that at runtime by collecting `window.mParticle.getVersion()`.